src/makeRobotAccountant.js

```diff
@@ -6,7 +6,24 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let numbersSum = 0;
+  let countCalls = 0;
+
+  const getSum = (firstNumber) => {
+    countCalls++;
+
+    return (secondNumber) => {
+      if (countCalls > 3 && countCalls % 2 === 0) {
+        return 'Bzzz... Error!';
+      }
+
+      numbersSum = firstNumber + secondNumber;
+
+      return numbersSum;
```

Comment:
You can immediately return the result of the calculation without creating an additional variable
checklist # 4. You should use one variable for counting function calls

return firstNumber + secondNumber;